lib/logitech_receiver/descriptors.py

```diff
@@ -277,6 +277,12 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 				])
 _D('Couch Mouse M515', protocol=2.0, wpid='4007')
 _D('Wireless Mouse M525', protocol=2.0, wpid='4013')
+_D('Multi Device Silent Mouse M585/M590', codename='M585/M590', protocol=4.5, wpid='406B',
+               settings=[
+                   _FS.lowres_smooth_scroll(),
+                   _FS.pointer_speed(),
+               ],
+   )
```

Comment:
This file seems to use tabs, can you convert your indentation from spaces to tabs? Looks reasonable otherwise.

Reply:
resolved. @Lekensteyn may you revisit this PR?